fft/src/KokkosFFT_OpenMP_plans.hpp

```diff
@@ -12,7 +12,7 @@
 namespace KokkosFFT {
 namespace Impl {
 template <typename ExecutionSpace, typename T>
-void _init_threads(const ExecutionSpace& exec_space) {
+void _init_threads([[maybe_unused]] const ExecutionSpace& exec_space) {
```

Comment:
Perhaps this file should be renamed CPU or host?

Reply:
Makes sense. Maybe better to call it Host? I will fix this in another PR.